src/app/core/services/api/fake-backend/connector-fake-impl/data/test-assets.ts

```diff
@@ -121,6 +122,43 @@ This is a short description text that should be fully rendered without being **c
     dataModel: 'unspecified',
     geoReferenceMethod: 'Lat/Lon',
     transportMode: 'Rail',
+    sovereignLegalName: 'The Sovereign',
```

Comment:
funny, but maybe we should adjust this to be slightly more realistic, e.g. `Data Owning Company GmbH`